packages/forms/src/UIForm/fields/FieldTemplate/FieldTemplate.component.test.js

```diff
@@ -45,6 +45,34 @@ describe('FieldTemplate', () => {
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
 
+	it('should render with hint', () => {
+		const tooltipContent = (
+			<span>Tooltip content, which helps to understand what is the purpose of this field</span>
+		);
+
+		// when
+		const wrapper = shallow(
+			<FieldTemplate
+				isValid
+				hint={{
+					overlayComponent: tooltipContent,
+					overlayPlacement: 'top',
+				}}
+				description={'My awesome description'}
+				descriptionId={'myAwesomeField-description'}
+				errorId={'myAwesomeField-error'}
+				errorMessage={'This is wrong o_o'}
+				id={'myAwesomeField'}
+				label={'My awesome label'}
+			>
+				<input id={'myAwesomeField'} />
+			</FieldTemplate>,
+		);
+
+		// then
+		expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
Can you select the title for the snapshot instead of the whole component ?

Reply:
yep, selected